.settings/org.eclipse.wst.jsdt.core.prefs

```diff
@@ -186,6 +186,7 @@ org.eclipse.wst.jsdt.core.formatter.insert_space_after_comma_in_superinterfaces=
 org.eclipse.wst.jsdt.core.formatter.insert_space_after_comma_in_type_arguments=insert
 org.eclipse.wst.jsdt.core.formatter.insert_space_after_comma_in_type_parameters=insert
 org.eclipse.wst.jsdt.core.formatter.insert_space_after_ellipsis=insert
+org.eclipse.wst.jsdt.core.formatter.insert_space_after_function_keyword=do not insert
```

Comment:
Is that related to the bug?

Reply:
no. for some reason, my eclipse automatically changed the setting while importing project

Reply:
then I guess it's cleaner if we don't include that in the PR :)